src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/ApplicationsTransformer.kt

```diff
@@ -41,7 +41,7 @@ class ApplicationsTransformer(
   private val cas1ApplicationUserDetailsTransformer: Cas1ApplicationUserDetailsTransformer,
 ) {
 
-  final val applicationStatuses = mapOf(
+  private final val applicationStatuses = mapOf(
```

Comment:
I've hit upon a new method for translating Enums from JPA to API and vice versa, and it'd be good to use that here. See https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1733

Reply:
Good idea, it makes much more sense for the translation to live as close as possible to the enum - I've made this change.